docs/release-notes/release-notes-0.14.2.md

```diff
@@ -87,10 +87,15 @@ connection from the watch-only node.
   exposed](https://github.com/lightningnetwork/lnd/pull/6146) inside
   WaitingCloseResp from calling `PendingChannels`.
 
+* [CustomCaveatCondition is now properly set]
+  (https://github.com/lightningnetwork/lnd/pull/6185) on
```

Comment:
This won't render correctly. You need to break inside the brackets, so for example:
```
* [CustomCaveatCondition is now properly
   set](https://github.com/lightningnetwork/lnd/pull/6185) on
```

Reply:
I didn't know the `(` had to be touching the `]`, thanks.